shared/actions/teams.js

```diff
@@ -908,22 +915,17 @@ function _updateTopic(action: TeamsGen.UpdateTopicPayload, state: TypedState) {
     identifyBehavior: RPCTypes.tlfKeysTLFIdentifyBehavior.chatGui,
   }
 
-  return Saga.all([
+  return Saga.sequentially([
     Saga.put(createIncrementWaiting(waitingKey)),
     Saga.call(RPCChatTypes.localPostHeadlineRpcPromise, param),
-    Saga.identity(
-      Saga.all([
-        Saga.put(createDecrementWaiting(waitingKey)),
-        Saga.put(TeamsGen.createGetChannels({teamname})),
```

Comment:
@mmaxim says these `getChannels` calls are unnecessary, and same below

Reply:
this only holds if we're on the channel in question -- but we can just update the teams store ourselves